pkg/exporter/azureblob_exporter.go

```diff
@@ -33,23 +29,18 @@ func NewAzureBlobExporter(creationTime, hostname string) *AzureBlobExporter {
 }
 
 func createContainerURL() (azblob.ContainerURL, error) {
-	APIServerFQDN, err := utils.GetAPIServerFQDN()
-	if err != nil {
-		return azblob.ContainerURL{}, err
-	}
+	accountName := os.Getenv("AZURE_BLOB_ACCOUNT_NAME")
+	sasKey := os.Getenv("AZURE_BLOB_SAS_KEY")
+	containerName := os.Getenv("AZURE_BLOB_CONTAINER_NAME")
 
-	containerName := strings.Replace(APIServerFQDN, ".", "-", -1)
-	containerLen := strings.Index(containerName, "-hcp-")
-	if containerLen == -1 {
-		containerLen = maxContainerNameLength
+	if accountName == "" || sasKey == "" || containerName == "" {
```

Comment:
🐛 Please add check for `containerName` length. `3 to 63` is the min and max - otherwise it will fail and in that case how will we handle that scenario? https://docs.microsoft.com/en-us/rest/api/storageservices/naming-and-referencing-containers--blobs--and-metadata#container-names 

There are bunch of stuff removed and will sit under consuming tool which looks cool.

Reply:
hmmm sure but in the same logic we could also check the accountname format, the saskey format etc ... I'd rather not do checks here. Anyway the user will eventually have an error from the service

Reply:
💡This logic was already pre-existing in this tool, if we keep that length as it is it and do not remove we take pain from other consuming tool managing their own version of this?